byway/src/main/webapp/routepage.js

```diff
@@ -159,19 +164,39 @@ function clearRoute(){
 /** Get trip info from datastore onload */
 function getRouteOnload(){
   clearRoute();
-  fetch('/api/stop')
+  fetch(configureTripKeyForPath(tripKey, "/api/stop"))
   .then(response => response.json())
-  .then((trip) => {
+  .then(async (trip) => {
```

Comment:
Good use of some the async functionality we discussed this week!

Reply:
Thanks! It helped out because I wasn't sure if this was proper.